tools/IoTEdgeDevOps/DevOpsLib/VstsModels/VstsBuild.cs

```diff
@@ -41,9 +46,13 @@ public class VstsBuild : IEquatable<VstsBuild>
         [JsonProperty("lastChangedDate")]
         public DateTime LastChangedDate { get; set; }
 
+        [JsonProperty("requestedBy")]
+        public Dictionary<String, Object> RequestedBy { get; set; }
```

Comment:
Why is this a Dictionary? I see in one of the modifications below that "displayName" is a key -- what are some other keys?

Reply:
This is deserializing a json key -> dictionary. There are other keys besides display name within the dict which are other contextual information about how it is requested. I don't remember which ones.

 From me playing around with it, `displayName` is always there and set to a specific name if it is requested.